docs/docsite/rst/scenario_guides/guides.rst

```diff
@@ -14,9 +14,3 @@ We are migrating these guides into collections. Please update your links for the
 
 :ref:`ansible_collections.amazon.aws.docsite.aws_intro`
```

Comment:
Out of curiosity (and not really related to this PR): should this list stay as it is, or should other guides of collections contained in Ansible be added here?

Reply:
Yeah was thinking about that as well. Let's discuss on the forum topic.